README.md

```diff
@@ -1,41 +1,64 @@
 # scripts
 
 Repository for internal scripts.
-Primarily, these scripts used on omnibus voting.
+Primarily, these scripts are used on omnibus voting.
 
 ## Network setup
-#### Mainnet-fork (default)
 
-By default, all scripts run in mainnet fork mode (don't forget to review fork param at brownie config. 
+In a typical weekly omnibus workflow, you need only the `mainnet-fork` and `mainnet` networks. In the case of a large test campaign on Lido upgrades, it also could be useful to go with `görli` and `görli-fork` testnets first.
 
-To run tests you need to set the additional brownie commandline params: `--network mainnet-fork` and `-s`. The first one needed for etherscan addresses resolving and the second one to reveal stdout. Also, define the following env variables: 
+This section covers network configuration params and their usage.
 
+### Environment variables setup
+
+Despite the chosen network you always need to set the following var:
+```bash
+export WEB3_INFURA_PROJECT_ID=<infura_api_key>
+```
+
+To start a new vote please provide the `DEPLOYER` brownie account name (wallet):
+```bash
+export DEPLOYER=<brownie_wallet_name>
+```
+
+To run tests with a contract name resolution guided by the Etherscan you should provide the etherscan API token:
 ```bash
-export ETHERSCAN_TOKEN=<etherscan_key>
-export WEB3_INFURA_PROJECT_ID=<infura_key>
+export ETHERSCAN_TOKEN=<etherscan_api_key>
 ```
 
-###### Running tests inside mainnet fork
+### Command-line arguments requirements
 
-* To re-use already created `vote_id` you can pass the `OMNIBUS_VOTE_ID` environment variable (e.g. `OMNIBUS_VOTE_ID=104`).
-* To forcely bypass etherscan contract and event names decoding set the `OMNIBUS_BYPASS_EVENTS_DECODING` environment variable to `1`.
-#### Mainnet
+Always pass network name explicitly with `--network {network-name}` brownie command-line arguments for both vote and tests scripts.
 
-To run scripts on actually mainnet you need to add param `--network mainnet` to the end of the command and set the following env variables:
+To reveal a full test output pass the `-s` flag when running test scripts with `brownie test`
+#### Fork-mode development networks
 
+By default, you should start testing process in a network fork mode (don't forget to review brownie fork params). The Mainnet-fork mode exists in brownie by default. To add the Görli Testnet fork mode please run once the following command (to setup and save new network):
 ```bash
-export DEPLOYER=<brownie_wallet_name>
-export WEB3_INFURA_PROJECT_ID=<infura_key>
+brownie networks add "Development" goerli-fork host=http://127.0.0.1 cmd=ganache-cli port=8545 gas_limit=12000000 fork=https://goerli.infura.io/v3/${WEB3_INFURA_PROJECT_ID} chain_id=5 mnemonic=brownie accounts=10
```

Comment:
Can it be added directly to brownie-config.yml?

Reply:
Nope, I wish it was so, but the `goerli-fork` testnet needs to be added explicitly first into the brownie networks list